transport/src/main/java/io/netty/channel/ChannelOutboundBuffer.java

```diff
@@ -220,16 +220,28 @@ public Object current() {
         return entry.msg;
     }
 
+    /**
+     * Return the current message flush progress.
+     * @return return {@code 0} if nothing was flushed before
```

Comment:
...if nothing was flushed before for the current message or there is no current message

Reply:
Done